wagtail/admin/compare.py

```diff
@@ -14,7 +14,6 @@
 from wagtail.utils.registry import ModelFieldRegistry
 from wagtail.utils.text import text_from_html
 
-
```

Comment:
to keep the old behaviour of 2 lines after imports, you could add

```
[isort]
lines-after-imports = 2
```

Reply:
Looking back on code from the 2.x-3.x branches, I can't see any sign that we followed that as a standard - we do have two lines before class / function definitions, but as far as I can see it's only been one line above assignments like this one.